tests/e2e/performance/csi_tests/test_pvc_snapshot_performance.py

```diff
@@ -166,6 +198,29 @@ def test_pvc_snapshot_performance(self, teardown_factory, pvc_size):
 
         all_results = []
 
+        self.full_log_path = get_full_test_logs_path(cname=self)
+        self.results_path = get_full_test_logs_path(cname=self)
+        self.full_log_path += f"--PVC-SIZE-{pvc_size}"
+        log.info(f"Logs file path name is : {self.full_log_path}")
+
+        # Produce ES report
+        # Collecting environment information
+        self.get_env_info()
+
+        # Initialize the results doc file.
+        self.full_results = self.init_full_results(
+            ResultsAnalyse(
+                self.uuid,
+                self.crd_data,
+                self.full_log_path,
+                "pvc_snapshot_perf",
+            )
+        )
+        self.full_results.add_key("pvc_size", pvc_size + "GiB")
+        self.full_results.all_results["creation_time"] = []
+        self.full_results.all_results["creation_speed"] = []
+        self.full_results.all_results["restore_time"] = []
+        self.full_results.all_results["restore_speed"] = []
```

Comment:
I think you should also add to the results the interface . 

full_results.add_key("interface", self.interface)